### ISSUE TITLE
[XamarinProfiler-MAC]: Accessibility: Color Contrast: MAS1.4.3: Non-active tabs (Inspect/Stack trace/Paths to roots) among them are not having color contrast ratio of 4.5:1 in Allocations/Call Tree/Snapshots/Samples windows of both Allocations/Time Profiler.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#416]
### Steps to Reproduce

1. Follow the pre-requisite to launch the xamarin profiler application.
2. Select the Available instruments to "Allocations/Time Profiler" and invoke next button. Invoke Start Profiling button.
3.This will invoke "Allocations/Time Profiler" Profiling page.
4. Verify the color contrast ratio of non-active tabs among (Inspect/Stack trace/Paths to roots) in  Allocations/Call Tree/Snapshots/Samples windows of both Allocations/Time Profiler.

### Pre-Requisite:
"1. Install Xamarin Profiler.
2. Launch Visual Studio.
3. Create a project and build the solution.
4. Navigate to Run Menu Tab and Invoke it. Navigate to "Start Profiling" option and invoke it.
5. This will invokes "Xamarin Profiler" application."

### Expected Behavior
Color contrast ratio should be minimum 4.5:1
### Actual Behavior
Color contrast ratio is less than 4.5:1 i.e, 3.7:1
See the screenshots attached
### User Impact
Low vision user will feel difficult in seeing the text
### Screenshot
![6 allocations](https://user-images.githubusercontent.com/22956130/41591226-bcc825e4-73d6-11e8-99e2-f36469cd4804.png)
![6 time profiler](https://user-images.githubusercontent.com/22956130/41591237-c5dd088e-73d6-11e8-8bce-75ef58ffe3f3.png)

### Environment
macOS High Sierra OSX version: 10.13.4
Visual Studio for Mac Enterprise version: 7.5.2(build 40)
Xamarin Profiler version: 1.6.3-400

**Comment by WiproPSAccTest** [comment]
Please add the following labels to this bug:
A11yMAS
Accessibility
MAS1.4.3
Xamarin-Profiler-MAC
Wipro
